demo/src/app/components/toast/overview/toast-overview.component.ts

```diff
@@ -42,9 +42,15 @@ export class NgbdToastOverviewComponent {
   APP_TOAST_SERVICE = Snippet({
     lang: 'typescript',
     code: `
+      export interface ToastMessage {
+        header: string;
+        body: string;
+        delay?: number;
+      }
+      
       @Injectable({ providedIn: 'root' })
       export class AppToastService {
-        toasts: any[] = [];
+        toasts: ToastMessage[] = [];
```

Comment:
I wouldn't name it `ToastMessage` as is contains `delay` too, but I have no good alternatives. `ToastData`, `ToastInfo`, just `Toast` maybe ?

Reply:
Sounds reasonable, I think I like `ToastInfo`. Let's go with that then :)